dynamiqs/qarrays/dense_qarray.py

```diff
@@ -62,6 +62,7 @@ def ptrace(self, *keep: int) -> QArray:
 
         return ptrace(self.data, keep, self.dims)
 
+    @partial(jax.jit, static_argnums=(1,))
```

Comment:
Why doing this here? We don't jit functions by default for now. Same question for the other `powm` implementations.

Reply:
Yes, I initially thought that adding this here would force the argument to be static. But if there's an outer jax.jit call (just like in the tests), then this is essentially ignored and still fails. I'll remove it.